projects/storefrontlib/src/shared/services/trigger/config/default-trigger-config.ts

```diff
@@ -3,11 +3,11 @@ import { OutletPosition } from '../../../../cms-structure/outlet/index';
 
 export const DEFAULT_TRIGGER_CONFIG: TriggerConfig = {
   trigger: {
-    asm: {
+    ASM: {
       outlet: 'cx-storefront',
       position: OutletPosition.BEFORE,
```

Comment:
I'd prefer a default position for outlets. If you agree, and if it's BEFORE, we can leave it out here.